src/bento-core/GlobalSearch/SearchResults/components/Cards/StudyCard.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Grid, withStyles } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import { prepareLinks } from 'bento-components';
+import PropertyItem from './PropertyItem';
+
+const CARD_PROPERTIES = [
+  {
+    label: 'Program ID',
+    dataField: 'program_id',
+    link: '/program/{program_id}',
+  },
+  {
+    label: 'Study Name',
+    dataField: 'study_name',
+  },
+  {
+    label: 'Study Type',
+    dataField: 'study_type',
+  },
+];
+
+const StudyCard = ({ data, classes, index }) => {
+  const propertiesWithLinks = prepareLinks(CARD_PROPERTIES, data);
+
+  return (
+    <Grid item container className={classes.card}>
+      <Grid item xs={1} className={classes.indexContainer}>
+        {index + 1 }
+      </Grid>
+      <Grid item xs={11} className={classes.propertyContainer}>
+        <div>
+          <span className={classes.detailContainerHeader}>STUDY</span>
+          <span className={classes.cardTitle}>
+            <Link to={`/arm/${data.study_code}`} className={classes.cardTitle}>
+              {data.study_id}
+            </Link>
+          </span>
+        </div>
+        {propertiesWithLinks.map((prop, idx) => (
+          <PropertyItem
+            index={idx}
+            label={prop.label}
+            value={data[prop.dataField]}
+            link={prop.link}
+          />
+        ))}
+      </Grid>
+    </Grid>
+  );
+};
+
+const styles = (theme) => ({
+  indexContainer: {
+    padding: '18px 0px 18px 18px',
+    color: '#747474',
+    fontFamily: 'Inter',
+    fontSize: '13px',
+  },
+  propertyContainer: {
+    padding: '16px 16px 16px 0px',
+    borderBottom: '2px solid #E7EEF5',
+  },
+  cardTitle: {
+    color: theme.palette.text.link,
+    textDecoration: 'none',
+    fontSize: '16px',
+    fontFamily: 'Nunito',
+    paddingLeft: '9px',
+    verticalAlign: 'middle',
+  },
+  detailContainerHeader: {
+    textTransform: 'uppercase',
+    padding: '2px 8px',
+    backgroundColor: '#DBDBDB',
+    color: '#000000',
+    fontFamily: 'Nunito',
+    fontSize: '12px',
+    fontWeight: '400',
+    letterSpacing: '0.9px',
+    verticalAlign: 'middle',
+    borderRadius: '4px',
+  },
+});
+
+export default withStyles(styles, { withTheme: true })(StudyCard);
```

Comment:
@amattu2 Do Is it possible to pass custom styles to this components?

Reply:
It's possible to override the styles for cards or add new card types without modifying the Global Search component in Bento Core.

If we add a new style class to the Search Page `styles.js` file 

https://github.com/CBIIT/bento-frontend/blob/b59a04953c23b0974963a8e88a940cdbfb8648ab/src/pages/search/styles.js#L1

We can override the default styles by updating the tab configuration

https://github.com/CBIIT/bento-frontend/blob/b59a04953c23b0974963a8e88a940cdbfb8648ab/src/pages/search/utils.js#L317-L320

This allows us to override styling for specific tabs (e.g. The study card).